arm/Microsoft.DesktopVirtualization/applicationgroups/.parameters/parameters.json

```diff
@@ -8,18 +8,44 @@
         "location": {
             "value": "westeurope"
         },
-        "appGroupType": {
+        "type": {
             "value": "RemoteApp"
         },
         "hostpoolName": {
             "value": "adp-sxx-az-avdhp-x-001"
         },
-        "appGroupFriendlyName": {
+        "friendlyName": {
             "value": "Remote Applications 1"
         },
         "appGroupDescription": {
             "value": "This is my first Remote Applications bundle"
         },
+        "applications": {
+            "value": [
+                {
```

Comment:
Once implemented you established the Optional vs. Mandatory parameters please also add one instance here that only provides the bare minimum. This way we can test that the child-resource parameter work properly